### PR TITLE
Blockchain Module: New Readme Format for Singapore

### PR DESCRIPTION
**Pull request by JoshOrndorff**
This PR revises the module 3 content in the direction requested in #982 but is made directly against main to better preserve the history of the files.

If this pr is accepted it can also be merged into #982 for anyone following that branch.

Tasklist:
- [x] Change format of readme
- [x] Change file names for lessons
- [x] Write learning outcomes 
- [x] Move start a blockchain info to dedicated repo
- [x] prune unused images

### CODE REVIEW

**Review by wirednkod — CHANGES_REQUESTED** [review]
Please reopen this PR against `2024-singapore` repository instead of `main` - as track of changes are kept there.

**Comment by JoshOrndorff** [comment]
The CI is catching broken links but they are not from this PR, so I recommend merging once it is reviewed positively.

File: syllabus/3-Blockchain/README.md
[the remark this answers is not in the document]
Comment (JoshOrndorff) [reply syllabus/3-Blockchain/README.md:1]: It is not a dedicated lesson. I will introduce what smart contracts are and cover their history in the first two lessons. Then the topic of contracts will come up several times throughout the module organically as it usually does.

[the remark this answers is not in the document]
Comment (wirednkod) [reply syllabus/3-Blockchain/README.md:1]: Sounds great. Thank you Joshy

**Pull request merged**